internal/utils/templates/init_config.toml

```diff
@@ -99,6 +99,8 @@ enable_confirmations = false
 enable_signup = true
 # If enabled, users need to confirm their phone number before signing in.
 enable_confirmations = false
+# Template for sending OTP to users
+template = "Your code is {{ .Code }} ."
```

Comment:
**suggestion**:
As this is parsed as if it was an actual go template, it is failing because it can't find the reference to ".Code".
I would change this to

template = "Your code is {{ `{{.Code }}` }} ."
So the output in config.yaml is

template = "Your code is {{ `{{.Code }}` }} ."

Reply:
Made the changes, not sure how I can test it out

Reply:
@jibin2706 
To begin with, the command `go run . init` wont fail.
Then, to manually test the deployment, after `go run . start` you can do something like:

```bash
# store gotrue container id
export CONTAINER=$(docker container ls | grep gotrue | awk '{print $1}')

# check what is in the env
docker container exec $CONTAINER printenv | grep GOTRUE_SMS_TEMPLATE=
```
That will output the environment that was injected into the container. You can then check if the value is the one you are expecting.

Reply:
thanks @LautaroJayat, working as expected